### PR TITLE
Support limits the max namespaces per tenant

### LINKED ISSUE #8224: Support limits the max namespaces per tenant
**Is your feature request related to a problem? Please describe.**
Support limits the max namespaces per tenant of the Pulsar cluster. When the namespaces reach the max namespaces of the tenant, the broker should reject the new namespace request.

**Describe the solution you'd like**
Add `maxNamespacePerTenant=0` in the broker.conf and don't limit by default.

### PR DESCRIPTION
**Pull request by 315157973**
Fixes #8224
### Motivation
Support limits the max namespaces per tenant of the Pulsar cluster. When the namespaces reach the max namespaces of the tenant, the broker should reject the new namespace request.

### Modifications
Add maxNamespacePerTenant=0 in the broker.conf and don't limit by default.

### Verifying this change
AdminApiTest2#testMaxNamespacePerTenant

### CODE REVIEW

File: conf/broker.conf [hunk 1]
```diff
@@ -201,6 +201,10 @@ brokerDeduplicationProducerInactivityTimeoutMinutes=360
 # value will be used as the default
 defaultNumberOfNamespaceBundles=4
 
+# The maximum number of namespaces that each tenant can create
+# This configuration is not precise control, in a concurrent scenario, the threshold will be exceeded
+maxNamespacePerTenant=0
```
Comment (codelipenghui) [inline conf/broker.conf:206]:
```suggestion
maxNamespacesPerTenant=0
```

Comment (codelipenghui) [reply conf/broker.conf:206]: Please check all.

**Comment by Huanli-Meng** [comment]
docs should be updated because the broker.config file is updated. So i add a doc-required label.

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**